coralproject/talk
# Permalink by Query Param instead of Hash

## Permalink by Query Param instead of Hash

## What does this PR do?
- Uses `commentId` query param instead of hash
- Solves error while using wrong commentId

## How do I test this PR?

- Go to the Stream 
- Select a comment
- Check the permalink (you should have `talk-plugin-permalink` enabled)
- Open that in another tab
- Replace that commentId by a fake one
